src/modules/enrollment/hooks/useEnrollmentSummaryDialog.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Button,
+  DialogActions,
+  DialogContent,
+  DialogProps,
+  DialogTitle,
+} from '@mui/material';
+import { useCallback, useMemo, useState } from 'react';
+
+import CommonDialog from '@/components/elements/CommonDialog';
+import GenericTable from '@/components/elements/table/GenericTable';
+import { ColumnDef } from '@/components/elements/table/types';
+import HmisEnum from '@/modules/hmis/components/HmisEnum';
+import { entryExitRange, parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import { Routes } from '@/routes/routes';
+import { HmisEnums } from '@/types/gqlEnums';
+import { EnrollmentSummaryFieldsFragment, ProjectType } from '@/types/gqlTypes';
+import generateSafePath from '@/utils/generateSafePath';
+
+const baseColumns: ColumnDef<EnrollmentSummaryFieldsFragment>[] = [
+  {
+    header: 'Enrollment Period',
+    render: (e) => entryExitRange(e),
+  },
+  {
+    header: 'Project',
+    linkTreatment: true,
+    render: (e) => e.projectName,
+  },
+  {
+    header: 'Project Type',
+    render: (e) => (
+      <HmisEnum value={e.projectType} enumMap={HmisEnums.ProjectType} />
+    ),
+  },
+  {
+    header: 'Move in Date',
+    render: (e) => {
+      const projectTypes = [
+        ProjectType.Ph,
+        ProjectType.Oph,
+        ProjectType.Th,
+        ProjectType.Psh,
+      ];
+
+      if (projectTypes.includes(e.projectType))
+        return parseAndFormatDate(e.moveInDate);
+
+      return null;
+    },
```

Comment:
Lets simplify this to just render the move-in date. The limitation to PH projects is default for collection, but lets just show it if we have it.
```suggestion
    render: (e) => parseAndFormatDate(e.moveInDate)
```